models/mail.go

```diff
@@ -160,7 +160,7 @@ func composeIssueMessage(issue *Issue, doer *User, tplName base.TplName, tos []s
 	if err != nil {
 		log.Error(3, "HTMLString (%s): %v", tplName, err)
 	}
-	msg := mailer.NewMessageFrom(tos, fmt.Sprintf(`"%s" <%s>`, doer.DisplayName(), setting.MailService.User), subject, content)
+	msg := mailer.NewMessageFrom(tos, fmt.Sprintf(`"%s" <%s>`, doer.DisplayName(), setting.MailService.From), subject, content)
```

Comment:
With 6e0a1a93d99ecc9549d4609d116dbc0f64023aab it would not be
a problem for the `FROM` to be `name <email>`.

BTW, according to the `Config Cheat Sheet`, `USER` could also not
be an email (it says "*usually* your e-mail address").